pkg/constants/constants.go

```diff
@@ -446,6 +449,16 @@ const (
 	SupportedModelMLFlow      = "mlflow"
 )
 
+// opendatahub rawDeployment Auth
+const (
+	OauthProxyImage                 = "registry.redhat.io/openshift4/ose-oauth-proxy@sha256:234af927030921ab8f7333f61f967b4b4dee37a1b3cf85689e9e63240dd62800"
+	OauthProxyPort                  = 8443
+	OauthProxyResourceMemoryLimit   = "256Mi"
+	OauthProxyResourceCPULimit      = "100m"
+	OauthProxyResourceMemoryRequest = "256Mi"
+	OauthProxyResourceCPURequest    = "100m"
```

Comment:
set the defaults as suggested here: https://github.com/opendatahub-io/kserve/pull/419/files#r1840373813